app/controllers/shared/api.php

```diff
@@ -511,6 +520,8 @@
         $dbForProject
             ->on(Database::EVENT_DOCUMENT_CREATE, 'calculate-usage', fn ($event, $document) => $usageDatabaseListener($event, $document, $queueForStatsUsage))
             ->on(Database::EVENT_DOCUMENT_DELETE, 'calculate-usage', fn ($event, $document) => $usageDatabaseListener($event, $document, $queueForStatsUsage))
+            ->on(Database::EVENT_DOCUMENTS_CREATE, 'calculate-usage', fn ($event, $document) => $usageDatabaseListener($event, $document, $queueForStatsUsage))
+            ->on(Database::EVENT_DOCUMENTS_DELETE, 'calculate-usage', fn ($event, $document) => $usageDatabaseListener($event, $document, $queueForStatsUsage))
```

Comment:
Calculate usage doesn't seem like an every but like a function, are we sure there is no better pattern to follow here?

Reply:
`calculate-usage` is the listener name, while `Database::EVENT_DOCUMENTS_CREATE` is the event, we do it this way so that you can add and remove multiple named listeners per event